DetourCrowd/Include/DetourCrowd.h

```diff
@@ -143,9 +143,9 @@ struct dtCrowdAgent
 	float desiredSpeed;
 
 	float npos[3];		///< The current agent position. [(x, y, z)]
-	float disp[3];
-	float dvel[3];		///< The desired velocity of the agent. [(x, y, z)]
-	float nvel[3];
+	float disp[3];          ///< "disp" is just something used during collision detection
```

Comment:
This could be better described.  We should indicate that it's used only in collision detection in DetourCrowd::update, but also indicate what it actually means in that context, rather than saying it's "just something".